scripts/dev_setup.sh

```diff
@@ -209,6 +210,26 @@ function install_awscli {
   aws --version
 }
 
+function install_trivy {
+  VERSION=$(trivy --version | head -1)
+  if [[ "$VERSION" != "Version: ${TRIVY_VERSION}" ]]; then
+    if [[ $(uname -s) == "Darwin" ]]; then
+      install_pkg aquasecurity/trivy/trivy brew
+    else
+      install_pkg wget "$PACKAGE_MANAGER"
+      install_pkg apt-transport-https "$PACKAGE_MANAGER"
```

Comment:
pretty sure this is debian/ubuntu specific and will break in alpine/elsewhere.   I recently cleaned up sccache to be ready for alpine.  🤷‍♂️

Reply:
let me double check the test results